src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import utils.Validator;
+
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        Validator.checkLottoSizeNumberInput(numbers);
+        Validator.checkLottoDuplicatedNumberInput(numbers);
+    }
+
+    // TODO: 추가 기능 구현
+    public List<Integer> getLottoNumbers() {
+        return numbers;
+    }
+
+    public int match(List<Integer> list) {
+        int count = 0;
+        for (Integer integer : list) {
+            if (numbers.contains(integer)) {
+                count++;
+            }
+        }
+        return count;
```

Comment:
```suggestion
    public int match(List<Integer> list) {
        return (int) list.stream()
                    .filter(numbers::contains)
                    .count();
    }
```
stream() 으로 변경하면 코드 가독성이 더 올라갈듯 해요!

Reply:
덕분에 스트림처리에 대해서 알아보고 있습니다!
감사합니다 :-)